dora/core/server/worker/src/main/java/alluxio/worker/dora/DoraMetaManager.java

```diff
@@ -109,20 +109,14 @@ protected UnderFileSystem getUfsInstance(String ufsUriStr) {
   /**
    * Gets file meta from UFS.
    * @param path the full ufs path
-   * @return the file status, or empty optional if not found
+   * @param ufs the underFileSystem
+   * @return the file ufsStatus, or empty optional if not found
    */
-  public Optional<FileStatus> getFromUfs(String path) throws IOException {
+  public Optional<UfsStatus> getFromUfs(String path, UnderFileSystem ufs) throws IOException {
```

Comment:
Actually alluxio is able to get ufs from an instance. I don't really understand why add ufs into paramter list.